exercises/practice/flatten-array/.meta/Example.roc

```diff
@@ -1,5 +1,6 @@
 module [flatten]
 
+flatten : [NestedArray (List NestedValue), Null, Value I64] as NestedValue -> List I64
```

Comment:
I think it would be better to extract a type alias for this (assuming it doesn't cause a compiler crash) which would remove the need for `as`.

Reply:
Great suggestion, thanks, done.